### PR TITLE
📖 Add docs for inline bookend config

### LINKED ISSUE #19200: Add documentation for inline config in amp-story-bookend
Now that we've introduced the ability to add inline JSON configs for the bookend (#19171), we have to add documentation to let amp-story developers know they can do this.

### PR DESCRIPTION
**Pull request by Enriqe**
Realized we never wrote docs about including the bookend inline for #15826.

This lets publishers know they can specify the bookend either from a `src` or by placing it inline in the markup.

Closes #19200

### CODE REVIEW

File: extensions/amp-story/amp-story.md [hunk 1]
```diff
@@ -934,13 +934,36 @@ To use it, include an `<amp-story-bookend>` tag as the child of your `<amp-story
   <amp-story-page id="cover">
     ...
   </amp-story-page>
-  <!-- `src` and `layout=display` are required. -->
+  <!-- `layout=display` is required. -->
```
Comment (gmajoulet) [inline extensions/amp-story/amp-story.md]: s/display/nodisplay

File: extensions/amp-story/amp-story.md [hunk 2]
```diff
@@ -934,13 +934,36 @@ To use it, include an `<amp-story-bookend>` tag as the child of your `<amp-story
...
   <amp-story-bookend src="bookendv1.json" layout=nodisplay>
   </amp-story-bookend>
 <amp-story>
 ```
 
-Next, you must create a JSON file where you can customize the bookend. The overall structure of the config looks like so:
+If you don't want to fetch the bookend config from a source, you can also specify it inline:
```
Comment (gmajoulet) [inline extensions/amp-story/amp-story.md]: Nit: s/config/configuration s/source/server

Also, I feel like with this wording, this section would only make sense if after the original one.

File: extensions/amp-story/amp-story.md [hunk 3]
```diff
@@ -934,13 +934,36 @@ To use it, include an `<amp-story-bookend>` tag as the child of your `<amp-story
...
+
+```html
+<amp-story standalone>
+  <amp-story-page id="cover">
```
Comment (gmajoulet) [inline extensions/amp-story/amp-story.md]: Nit: maybe we can save a few lines by not repeating `amp-story-page` here, and have the `shareProviders` + `components` on one line each. It'd make sense if it's the second example

**Comment by Enriqe** [comment]
@CrystalOnScript does this look good to you? :)

**Pull request merged**